src/containers/Transactions/Simple/NFTokenCancelOffer.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Account from '../../shared/components/Account';
+import Currency from '../../shared/components/Currency';
+
+const NFTokenCancelOffer = props => {
+  const {
+    data: {
+      instructions: { cancelledOffers },
+    },
+  } = props;
+
+  return (
+    <>
+      {cancelledOffers.map(offer => (
+        <>
+          <div className="row flex-wrap">
+            <div className="label">Offer ID</div>
+            <div className="value">
+              <div className="dt">{offer.offerID}</div>
+            </div>
+          </div>
+          <div className="row flex-wrap">
+            <div className="label">Token ID</div>
+            <div className="value">
+              <div className="dt">{offer.tokenID}</div>
+            </div>
+          </div>
+          <div className="row">
+            <div className="label">Amount</div>
+            <div className="value">
+              {offer.amount.amount}
+              <Currency currency={offer.amount.currency} issuer={offer.amount.issuer} />
```

Comment:
nit: would be cleaner to destructure `offer` object at the top of the function and delete `offer.` prefixes.